android/src/test/java/com/browserstack/BrowserStackJBehaveRunner.java

```diff
@@ -0,0 +1,84 @@
+package com.browserstack;
+
+import java.io.File;
+import java.io.InputStream;
+import java.net.URL;
+import java.nio.file.Files;
+import java.util.*;
+import java.lang.reflect.Constructor;
+
+import io.appium.java_client.android.AndroidDriver;
+import org.jbehave.core.embedder.Embedder;
+import org.junit.Test;
+import org.junit.After;
+import org.junit.Before;
+
+import org.openqa.selenium.MutableCapabilities;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.yaml.snakeyaml.Yaml;
+
+public class BrowserStackJBehaveRunner {
+
+    public WebDriver driver;
+    private static Map<String, Object> browserstackYamlMap;
+    private String userName;
+    private String accessKey;
+    public static final String USER_DIR = "user.dir";
+
+    @Before
+    public void setUp() throws Exception {
+        if (browserstackYamlMap == null) {
+            File file = new File(getUserDir() + "/browserstack.yml");
+            browserstackYamlMap = convertYamlFileToMap(file, new HashMap<>());
+        }
```

Comment:
Do we still want to have this? As now we have confidence on SDK?

Reply:
Currently keeping consistent with parent repo